cfgov/jinja2/v1/_includes/related-links.html

```diff
@@ -28,9 +28,14 @@ <h2 class="a-heading">
         </h2>
     </header>
 {%- if related_links_2 %}
-    <div class="content-l content-l__main content-l__large-gutters">
+    <div class="content-l
+                content-l__main
+                content-l__large-gutters">
         <div class="content-l_col content-l_col-1-2">
-            <ul class="m-list m-list__links u-mb0">
+            <ul class="m-list
+                       m-list__unstyled
+                       m-list__links
+                       u-mb0">
```

Comment:
`u-mb0` is unnecessary with our `last-child` rule: https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/enhancements/typography.less#L281-L285

(Also occurs once below.)